WcaOnRails/app/helpers/application_helper.rb

```diff
@@ -1,6 +1,8 @@
 # frozen_string_literal: true
 module ApplicationHelper
-  def full_title(page_title = '')
+  include MarkdownHelper # JFLY
```

Comment:
I'm surprised I had to do this. I did this because a view spec started to fail. From my research, I didn't see anything about helpers changing in Rails 5, so I'm wondering if this was just a test failure.

@jonatanklosko, @larspetrus, @timhabermaas, do you know anything about how loading of helpers is supposed to work in Rails?